src/common/LoadingListItem.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+
+import * as React from 'react';
+import { StyleSheet } from 'react-native';
+import { getRandomInt } from '../utils';
+import { ListItem as RNEListItem } from 'react-native-elements';
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: 'white',
+  },
+  subtitleContainerStyle: {
+    backgroundColor: '#eee',
+  },
+  subtitleStyle: {
+    color: '#eee',
+  },
+  titleContainerStyle: {
+    backgroundColor: '#eee',
+  },
+  titleStyle: {
+    color: '#eee',
+  },
+});
+
+type Props = {
+  containerStyle?: Object,
+};
+
+class LoadingListItem extends React.PureComponent<Props> {
+  render() {
+    return (
+      <RNEListItem
+        {...this.props}
+        hideChevron
+        subtitle="none"
+        subtitleContainerStyle={[
+          styles.subtitleContainerStyle,
+          { width: `${getRandomInt(60, 80)}%` },
```

Comment:
I think that these random values should probably be stored as state. I would think what you have is fine but there could be an edge case where the user sees a flash of content as the width is recalculated and changes size.